pkg/eventmonitor/config/config.go

```diff
@@ -30,21 +28,17 @@ type Config struct {
 
 	// ProcessConsumerEnabled defines if the process-agent wants to receive kernel events
 	ProcessConsumerEnabled bool
-
-	// NetworkConsumerEnabled defines if the network tracer system-probe module wants to receive kernel events
-	NetworkConsumerEnabled bool
 }
 
 // NewConfig creates a config for the event monitoring module
-func NewConfig(spConfig *sysconfigtypes.Config) *Config {
+func NewConfig() *Config {
 	return &Config{
 		// event server
 		SocketPath:       coreconfig.SystemProbe.GetString(join(evNS, "socket")),
 		EventServerBurst: coreconfig.SystemProbe.GetInt(join(evNS, "event_server.burst")),
 
 		// consumers
 		ProcessConsumerEnabled: getBool("process.enabled"),
-		NetworkConsumerEnabled: getBool("network_process.enabled") && spConfig.ModuleIsEnabled(config.NetworkTracerModule),
```

Comment:
would we want to maintain the ability to manually disable eventmonitor?

Reply:
Its still there since we check the same config in the network tracer module. We just don't check for it directly here; `events.Initialized()` should return `false` when this config is disabled.